src/nft/ZoraCreator1155Impl.sol

```diff
@@ -482,6 +482,10 @@ contract ZoraCreator1155Impl is
         return firstMinters[tokenId];
     }
 
+    function mintFee() external view returns (uint256) {
```

Comment:
Is/was this in the interface?

Reply:
If so, should add it back in. If not, as-is is fine.

Reply:
it looks like it was in the [IMintFeeManager]([url](https://github.com/ourzora/zora-1155-contracts/blob/main/src/interfaces/IMintFeeManager.sol#L9)) interface, which this no longer inherits from; should we still add it here?

Reply:
added in the most recent commit

Reply:
@oveddan does it affect the 165 of the entire contract? i think a test should catch this.